br/Makefile

```diff
@@ -99,7 +99,7 @@ unit_test_in_verify_ci: tools/bin/gotestsum tools/bin/gocov tools/bin/gocov-xml
 check: check-static
 
 check-static: tools/bin/golangci-lint
-	GO111MODULE=on CGO_ENABLED=0 tools/bin/golangci-lint run -v $$($(BR_PACKAGE_DIRECTORIES)) --config ../.golangci.yml
+	GO111MODULE=on CGO_ENABLED=0 tools/bin/golangci-lint run -v $$($(BR_PACKAGE_DIRECTORIES)) --config ../.golangci.yml --timeout 5m
```

Comment:
how about:
1. remove the `verify` Jinkens job
2. remove misleading target in `Makefile`

so that we can:
1. have a single source of truth scripts, which is in `.github/workflow`, to run and verify whether static checks are passed
2. make community able to define and optimize CI/CD itself without the help of PingCAP

Reply:
En...I am a little worried about that Github workflow cannot meet all the CI requirements.
In integration test (e.g., [tests/br_rawkv](https://github.com/tikv/migration/blob/2b13212b65a8fa4bb3eda96df36b6fc3460a35d4/br/tests/br_rawkv/run.sh#L20)), we need to setup a real cluster with pd-server & tikv-server. TiKV-CDC would have similar requirements.

How about let us fix this tiny issue now ? And later we will try to migrate all test cases to Github workflow.

Reply:
> we need to setup a real cluster with pd-server & tikv-server.

we can define runners (dockers) via github actions

> How about let us fix this tiny issue now 

sure. migrating to github action can be done in the future